packages/store/addon/-private/system/ds-model-store.ts

```diff
@@ -245,9 +245,13 @@ class Store extends CoreStore {
     }
   }
 
-  _attributesDefinitionFor(modelName: string) {
+  _attributesDefinitionFor(modelName: string, identifier?: StableRecordIdentifier) {
```

Comment:
Probably a silly question, but can't we use an identifier for the model name itself (e.g. when there is no `id`/`lid`)?

Reply:
Yup, going to propose an amendment to the RFC to slightly change the design to make it nicer to use

Reply:
Sounds good, we should probably try to do that before enabling the `CUSTOM_MODEL_CLASS` flag.